css/style.css

```diff
@@ -0,0 +1,337 @@
+  text-align: center;
+  color: #808080;
+  padding: 8px;
+  height: 24px;
+}
+
+.basit {
+  margin-top: 17px;
+  border-radius: 35px;
+  margin-right: 10px;
+  height: 27px;
+}
+
+.input-wrapper {
+  width: 100%;
+  margin-left: auto;
+  margin-right: auto;
+  align-content: center;
+}
+
+.input-list-item {
+  width: 80%;
+  padding: 0;
+  border: 0;
+}
+
+.input {
+  width: 80%;
+  height: 30px;
+  margin-right: 0;
+  margin-left: 20%;
+  margin-bottom: 20px;
+  margin-top: 20px;
+  border: 1px solid #808080;
+  border-right: none;
+}
+
+.input:focus {
+  border: 1px solid #000;
+}
+
+.botton-list-item {
+  width: 10%;
+  background-color: red;
+}
+
+.search-botton {
+  width: 60px;
+  height: 30px;
+  margin-left: 0%;
+  margin-right: 15%;
+  margin-top: 20px;
+  border-left: 1px solid #808080;
+  border: 1px solid #808080;
+}
+
+.main {
+  width: 100%;
+  display: flex;
+}
+
+.main-video {
+  width: 60%;
+  padding-left: 20px;
+}
+
+.iframe-video {
+  width: 100%;
+  height: 409px;
+  border: none;
+}
+
+.main-video-production {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 15px;
+}
+
+.main-video-list {
+  display: flex;
+  align-items: center;
+}
+
+.production {
+  background-color: #808080;
+  color: #fff;
+  text-decoration: none;
+  padding: 15px;
+  border-radius: 50px;
+  font-weight: bold;
+}
+
+.main-video-details {
+  width: 90%;
+}
+
+.production-details {
+  display: flex;
+  flex-direction: column;
+}
+
+.sub-button {
+  padding: 10px;
+  color: #fff;
+  background-color: #f00;
+  border: 2px solid #f00;
+  cursor: pointer;
+  font-weight: bolder;
+  border-radius: 2px;
+  font-family: arial, Verdana, Geneva, Tahoma, sans-serif;
+  margin-right: 20px;
+}
+
+.production-name {
+  font-size: 90%;
+  font-weight: bold;
+}
+
+.subscribers {
+  font-size: 80%;
+  color: #808080;
+}
+
+.video-description {
+  line-height: 2em;
+  font-size: 90%;
+}
+
+.latest-video {
+  width: 40%;
+  margin-left: 15px;
+}
+
+.latest-video-article {
+  display: flex;
+}
+
+.latest-video-side-show {
+  margin-top: 5px;
+  margin-right: 15px;
+}
+
+.latest-video-title {
+  font-size: 95%;
+}
+
+.latest-video-title-wrapper {
+  display: flex;
+  flex-direction: column;
+}
+
+.latest-video-details {
+  font-size: 80%;
+  color: #808080;
+}
+
+.latest-video-stats {
+  font-size: 75%;
+  color: #808080;
+}
+
+.trend {
+  font-size: 80%;
+  color: blue;
+  text-decoration: none;
+}
+
+.head {
+  font-weight: normal;
+  font-size: 130%;
+}
+
+.views {
+  font-size: 90%;
+  color: #808080;
+}
+
+.updates {
+  display: flex;
+  justify-content: space-between;
+}
+
+.cont {
+  display: flex;
+  flex-grow: 1;
+}
+
+.thumbs {
+  width: 24px;
+  height: 24px;
+  margin: 12px 10px;
+  fill: #808080;
+}
+
+.likes {
+  margin-top: 12px;
+  margin-right: 7px;
+  margin-left: 0;
+  color: #808080;
+}
+
+@media screen and (max-width: 993px) {
+  .main {
+    width: 100%;
+    display: block;
+  }
+
+  .main-video {
+    width: 100%;
+  }
```

Comment:
For future projects, I would recommend using CSS imports for media queries, see [here](https://developer.mozilla.org/en/docs/Web/CSS/@import) and [here](https://www.w3schools.com/cssref/pr_import_rule.asp) for more details.

CSS imports help with code splitting, among other benefits, it makes your code more organized.

Reply:
thanks for the recommendation.i will surely do that